.github/workflows/failpoint_test.yaml

```diff
@@ -5,7 +5,7 @@ jobs:
   test:
     strategy:
       matrix:
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, actuated-arm64-8cpu-32gb]
```

Comment:
@ahrtr iiuc, we dont use the `failpoint` so far, thats why i defaulted to `no` .. please correct me if i am wrong

Reply:
Suggest to keep the failpoint_test unchanged.

Reply:
done